packages/react-components/karma.conf.js

```diff
@@ -2,6 +2,8 @@
 
 process.env.CHROME_BIN = require('puppeteer').executablePath();
 
+process.env.TZ = 'Asia/Singapore';
```

Comment:
I suspect this might be the thing that fixes the tests.

Reply:
I was thinking the same thing, hence https://github.com/open-rmf/rmf-web/pull/681/commits/66a5338d6d3c22b129d4dc1e4a9daf82839cb4e8, however it still failed.

I kept this env variable in as it was mentioned as a general practice for CI.